packages/devtools/cli/src/base-command.ts

```diff
@@ -80,6 +82,34 @@ export abstract class BaseCommand extends Command {
     this.log('ok');
   }
 
+  async getStdin() {
```

Comment:
I thins https://nodejs.org/api/readline.html is the best way to read from stdin.
Move to util/readline.ts?

```
import readline from 'readline';

const rl = readline.createInterface({
  input: process.stdin,
  terminal: false
});

rl.on('line', (line) => {
    console.log(line);
});

rl.once('close', () => {
  // Done.
});
```